tests/test_cloudformation/test_cloudformation_stack_crud_boto3.py

```diff
@@ -1023,6 +1021,35 @@ def test_boto3_update_stack_fail_missing_new_parameter():
         )
 
 
+@mock_cloudformation
+def test_boto3_update_stack_fail_update_same_template_body():
+
+    name = "update_stack_with_previous_value"
+    cf_conn = boto3.client("cloudformation", region_name="us-east-1")
+    params = [
+        {"ParameterKey": "TagName", "ParameterValue": "foo"},
+        {"ParameterKey": "TagDescription", "ParameterValue": "bar"},
+    ]
+
+    cf_conn.create_stack(
+        StackName=name, TemplateBody=dummy_template_yaml_with_ref, Parameters=params,
+    )
+
+    with pytest.raises(ClientError) as exp:
+        cf_conn.update_stack(
+            StackName=name,
+            TemplateBody=dummy_template_yaml_with_ref,
+            Parameters=params,
+        )
+    exp_err = exp.value.response.get("Error")
+    exp_metadata = exp.value.response.get("ResponseMetadata")
+
+    exp_err.get("Code").should.match(r"ValidationError")
+    print(f"Message recieved: {exp_err.get('Message')}")
+    assert exp_err.get("Message") == f"Stack [{name}] already exists"
```

Comment:
```suggestion
    exp_err.get("Code").should.equal("ValidationError")
    exp_err.get("Message").should.equal(f"Stack [{name}] already exists")
```
This is more readable IMO

Reply:
Agreed, updated to match this.